daphne/src/taskprov.rs

```diff
@@ -227,7 +227,7 @@ impl VdafConfig {
         var: VdafTypeVar,
     ) -> Result<Self, DapAbort> {
         match (version, var) {
-            (.., VdafTypeVar::Prio2 { dimension }) => Ok(VdafConfig::Prio2 {
+            (_, VdafTypeVar::Prio2 { dimension }) => Ok(VdafConfig::Prio2 {
```

Comment:
Is this clippy's preference or yours? Or, is this just matching the internal repo?

Reply:
My preference and to align with the rest of the match and the internal repo